src/cli/node/with_node.rs

```diff
@@ -220,6 +231,39 @@ pub enum Command {
         #[structopt(long = "limit", allow_hyphen_values = true, default_value = "-50")]
         limit: i64,
     },
+    #[structopt(
+        name = "minerList",
+        alias = "getMiners",
+        about = "List block hashes with their miners and the total number of mined block by each address"
+    )]
+    GetMiners {
+        /// Socket address of the Witnet node to query
+        #[structopt(short = "n", long = "node")]
+        node: Option<SocketAddr>,
+        /// init
+        /// First epoch for which to return block hashes
+        /// If negative, return block hashes from the last n epochs
+        #[structopt(
+            long = "start",
+            alias = "i",
+            allow_hyphen_values = true,
+            default_value = "0"
+        )]
+        start: i64,
+        /// end
+        /// If negative, return the last n block hashes from this epoch range.
+        /// If zero, unlimited
+        #[structopt(
+            long = "end",
+            alias = "e",
+            allow_hyphen_values = true,
+            default_value = "-50"
```

Comment:
Am I understanding that the default behavior is "give me all the miners except the last 50"?

Reply:
give me the last 50 miners